qtensor/optimisation/Optimizer.py

```diff
@@ -107,16 +108,16 @@ def _get_kahyper_kwarge(self):
         kwargs = {'K': 2, 'eps': 0.1, 'seed': 2021, 'mode':0, 'objective':0} 
         return kwargs
         
-    def optimize(self, circ):
+    def optimize(self, tensor_net, tn):
```

Comment:
This will be incompatible with qtensor simulators. They will pass just the `qtensor.optimization.QtreeTensorNet` object. I suppose the problem is that the `tn` is generated from circuit. I think the function `circ2tn` can be changed to 
```python
def tn2tn(tn: qtensor.optimization.QtreeTensorNet):
```
 I know, I know.... there are two TNs that are different things but called the same.

Reply:
Yes, I understand. Will modify it.